pkg/types/jsonschema.go

```diff
@@ -3,11 +3,15 @@ package types
 import "encoding/json"
 
 type JSONSchema struct {
-	Property
+	Description string      `json:"description,omitempty"`
+	Type        string      `json:"type,omitempty"`
+	Ref         string      `json:"$ref,omitempty"`
+	Items       *JSONSchema `json:"items,omitempty"`
+	Enum        []string    `json:"enum,omitempty"`
 
 	ID         string                `json:"$id,omitempty"`
 	Title      string                `json:"title,omitempty"`
-	Properties map[string]Property   `json:"properties"`
+	Properties map[string]JSONSchema `json:"properties"`
```

Comment:
These are some fixes that I made to the JSONSchema. There was no need to have a separate `Property` struct.